tests/bugzilla.py

```diff
@@ -69,28 +93,30 @@ def setUpClass(cls):
             'url': 'http://localhost:27489/',
         })
         cls.bugzillaProvider.update_config(SimpleLoggerConfig)
+        t = Thread(target=cls.server.serve_forever)
+        t.start()
 
     @classmethod
     def tearDownClass(cls):
-        cls.server.server_close()
+        cls.server.shutdown()
```

Comment:
Should we do both a `shutdown` to break the http serving loop and a `server_close` to release the socket descriptor? (in that order)

Reply:
I guess so!